database/migrations/2023_01_03_185102_update_make_description_text_in_nests_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class UpdateMakeDescriptionTextInNestsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('nests', function (Blueprint $table) {
+            $table->text('description')->change();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('nests', function (Blueprint $table) {
+            $table->text('description')->change();
```

Comment:
Looks good to me. However you need to change the migration down function to change the type back to its original.

Reply:
ah yes, i see. should now be all right